digital_land/phase/map.py

```diff
@@ -6,8 +6,13 @@
 normalise_pattern = re.compile(r"[^a-z0-9-_]")
 
 
+def replace_underscores(name: str):
```

Comment:
this probably doesn't need an extra function, the function name is longer than the code it actually runs.

Reply:
I was being cautious as I'd seen normalise being used for things like urls.
I now appreciate that many modules have a normalize method, so the impact of changing map's normalise is probably localized to our need - affecting only column-names.
I'll extend normalise() as advised.

Reply:
I found that changing it to one function broke a lot of existing tests.
By leaving it as 2 functions, it works as we need and existing functionality needs.
I know it looks weird, but I feel that this is the best approach.

Reply:
the only tests that are failing are the ones you've written. I think the issue with them is that the mapping phase itself doesn't remove columns so the. old columns are left in